static/src/javascripts/lib/formInlineLabels.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import bean from 'bean';
+import $ from 'lib/$';
+import fastdom from 'fastdom';
+
+/**
+ * @param {String} type 'add' or 'remove'
+ * @param {Object} $el $ element
+ * @param {String} cssClass The class to add or remove
+ * @param {Function} testFunc A callback function that should return boolean describing whether we
+ * should perform the add or remove class
+ */
+
```

Comment:
Do we want to keep those comments around? Recently I deleted the ones I found, since the explanation is now somehow in Flow and I think the value is quite low, if we don't maintain them ..